SwrveSDK/SDK/Talk/SwrveMessageViewController.m

```diff
@@ -199,10 +202,20 @@ - (BOOL)shouldAutorotate
 }
 // ---------------
 
+- (void)orientationChanged:(NSNotification *)notification{
+    //After a given delay to allow for the correct orientation and view.frame to be calculated. Redraw the Message
+    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (u_int64_t)0.01 * NSEC_PER_SEC), dispatch_get_main_queue(), ^{
+        [self viewDidAppear:NO];
```

Comment:
Could you split the logic inside viewDidAppear to avoid calling it directly? Something like "redrawMessage"